include/image.mk

```diff
@@ -163,7 +163,13 @@ DTC_FLAGS += $(DTC_WARN_FLAGS)
 DTCO_FLAGS += $(DTC_WARN_FLAGS)
 
 define Image/pad-to
-	dd if=$(1) of=$(1).new bs=$(2) conv=sync
+	if [ $(2) -gt 1048576 ]; then \
```

Comment:
Is it worth it to add an additional check like `if [ $(2) -gt 1048576 -a $(shell expr $(2) % 1048576) -eq 0 ]`?

Reply:
Good question, actually it seems, that this copy&paste needs to be revisited and test, there might be more issues, like missing conversion of unit exponents, need to double check it.

Reply:
Yeah, this looks like it's relying on padding sizes over 1MiB to always be in multiples of 1MiB but I'm not sure if that's a valid assumption. I'm thinking about this a bit and maybe we could instead use the `head -c <size> <source file> /dev/zero` command instead.